src/aap_eda/core/enums.py

```diff
@@ -12,10 +12,10 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-from enum import Enum
+from enum import StrEnum
```

Comment:
we need to support python3.9

Reply:
BTW, the original check for a string type was predicated on the idea that an appropriate instance can be created based on a string which would throw an exception if not one of the defined enum values.  That, together with "just because something might be coercible to a string doesn't mean it's a good candidate for a comparand."

Interestingly (or not :-)) even 3.11's StrEnum doesn't handle byte objects for comparison (those being, in my experience, the things most likely to show up when you were expecting a string).

Reply:
But I think it is good to enforce the type, isn't?